flecs_ecs/src/core/world.rs

```diff
@@ -1852,17 +1848,46 @@ impl World {
     /// * [`World::cloned()`]
     /// * [`World::map()`]
     /// * [`World::try_map()`]
-    pub fn get<T: GetTupleTypeOperation>(&self, callback: impl for<'e> FnOnce(T::ActualType<'e>))
+    fn get<T: GetTupleTypeOperation>(
+        &self,
+        callback: impl for<'e> FnOnce(T::ActualType<'e>) -> Return,
+    ) -> Return
+    where
+        T::OnlyType: ComponentOrPairId;
+}
+
+impl<Return> WorldGet<Return> for World {
+    fn try_get<T: GetTupleTypeOperation>(
+        &self,
+        callback: impl for<'e> FnOnce(T::ActualType<'e>) -> Return,
+    ) -> Return::Wrapped
     where
         T::OnlyType: ComponentOrPairId,
+        Return: OptionOrZST,
     {
         let entity = EntityView::new_from(
             self,
             <<T::OnlyType as ComponentOrPairId>::CastType>::id(self),
         );
-        entity.get::<T>(callback);
+        entity.try_get::<T>(callback)
```

Comment:
why did this become try_get from get?

Reply:
It's an artefact of the diff algorithm used by github; it's getting confused by the code going from "try_get followed by get" to "try_get, get, try_get, get" due to the trait and impl